src/metabase/api/search.clj

```diff
@@ -97,25 +96,7 @@
                                 :search-native-query                 search_native_query
                                 :search-string                       q
                                 :table-db-id                         table_db_id
-                                :verified                            verified}))
-        duration             (- (System/currentTimeMillis) start-time)
-        has-advanced-filters (some some?
-                                   [models created_by created_at last_edited_by
-                                    last_edited_at search_native_query verified])]
-    (when (contains? #{"search-app" "search-bar" "command-palette"} context)
-      (snowplow/track-event! ::snowplow/new-search-query api/*current-user-id*
```

Comment:
Just so I understand — these events are being moved to the FE?

Reply:
Correct. We wanted to add click events to the search items, and apparently it's not ideal to have multiple sources of the events that we want to corollate